### DISCUSSION TITLE
With continuous wrapping, indent wrapped text at least as much as the first line

### DISCUSSION THREAD

**Discussion by julian-smith-artifex-com** [eteran/nedit-ng#351] [Ideas]
If a file contains long indented lines, i'd like to be able to tell nedit-ng to indent the wrapped portion of each line, when using continuous wrapping.

For example with this text:
```
      .. note:: A reserved fontname can be specified in any mixture of upper or
```

If we display it in a narrow nedit-ng window with continuous wrapping, it looks like this (with line numbers also enabled):
```
1       .. note:: A reserved
  fontname can be specified
  in any mixture of upper or
2
```

I'd like to be able to tell nedit-ng to display it like this instead:
```
1       .. note:: A reserved
        fontname can be specified
        in any mixture of upper or
2
```
I've seen at least one other editor that does this, and it makes a big difference when editing text with long indented lines, because it doesn't break up the view of the indentation in the file.

[Ideally we might want to indent slightly more than the first line, but that's a minor detail.]

**Comment by eteran** [comment]
Interesting, I'll look into it!
